### PR TITLE
Polish lobby faction tooltips

### PR DESCRIPTION
**Pull request by pchote**
While testing #13365 I noticed that the faction tooltip margins were wrong (copied from RA) for TD.  It turns out that `FactionTooltipLogic` is now completely redundant vs the standard button tooltip, so I have also removed that and made some other minor polish tweaks.

### CODE REVIEW

File: OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs [hunk 1]
```diff
@@ -118,7 +118,13 @@ public SlotDropDownOption(string title, string order, Func<bool> selected)
 				var flag = item.Get<ImageWidget>("FLAG");
 				flag.GetImageCollection = () => "flags";
 				flag.GetImageName = () => factionId;
-				item.GetTooltipText = () => faction.Description;
+
+				var displayName = faction.Description.SubstringBefore("\\n", StringComparison.Ordinal);
+				var displayDesc = faction.Description.SubstringAfter("\\n", StringComparison.Ordinal);
```
Comment (rob-v) [inline OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs]: displayName/Desc vs factionName/factionDescription  in SetupEditableFactionWidget

**Review by rob-v — APPROVED** [review]
👍 
small naming nit

**Comment by pchote** [comment]
Fixed and rebased.

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/269c50d6f48868b15f1d303feb974aba97d7f956...46e0a752d42a9a0a214dc073a59c7a64021e2bdc

**Pull request merged**